frontend/src/pages/profile/Profile.tsx

```diff
@@ -0,0 +1,110 @@
+import { Breadcrumbs, Button } from '@material-tailwind/react';
+import { useParams } from 'react-router-dom';
+import usePersonnel from '@/hooks/usePersonnel';
+import MemberDetails from './MemberDetails';
+
+function HorizontalLine() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      strokeWidth={1}
+      stroke="gray"
+      className="h-10 w-10"
+    >
+      <path id="triangle" d="M1 5 L0 35" />
+    </svg>
+  );
+}
+
+function BackIcon() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      strokeWidth={6}
+      stroke="currentColor"
+      className={'h-5 w-5 rotate-90'}
+    >
+      <path
+        strokeLinecap="round"
+        strokeLinejoin="round"
+        d="M19.5 8.25l-7.5 7.5-7.5-7.5"
+      />
+    </svg>
+  );
+}
+
+const Profile = () => {
+  const { personnelId } = useParams() as { personnelId: string };
+  const { personnel } = usePersonnel({ personnelId });
+
+  return (
+    <div className="h-screen pt-12 pb-24 bg-grayBackground">
+      {personnel && (
+        <div>
+          <Breadcrumbs placeholder={''} className="px-12">
+            <a href="/dashboard" className="text-linkBlue">
+              <div className="flex flex-row items-center">
+                <BackIcon />
+                <span className="pl-4 underline decoration-solid">
+                  Personnel (Dashboard)
+                </span>
+              </div>
+            </a>
+            <span className="font-bold">
+              {personnel.firstName} {personnel.lastName}
+            </span>
+          </Breadcrumbs>
+          <div className="pt-12">
```

Comment:
Would you be open to splitting this into "Profile Header" as it's own component? (So this page would basically be a container for three components: Breadcrumbs, Profile Header, Member Details? (or, Profile Details?))